packages/web-app-user-management/src/components/Groups/CreateGroupModal.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
```

Comment:
In theory this wouldn't need a wrapping div since it has only one root node anyways?